nftables.go

```diff
@@ -566,18 +554,23 @@ func objFromMsg(msg netlink.Message) (Obj, error) {
 	return nil, fmt.Errorf("malformed stateful object")
 }
 
-func (cc *Conn) getObj(o Obj, msgType uint16) ([]Obj, error) {
-	var conn *netlink.Conn
-	var err error
-	if cc.TestDial == nil {
-		conn, err = netlink.Dial(unix.NETLINK_NETFILTER, nil)
-	} else {
-		conn = nltest.Dial(cc.TestDial)
+func (cc *Conn) dialNetlink() (*netlink.Conn, error) {
+	if cc.TestDial != nil {
+		return nltest.Dial(cc.TestDial), nil
+	}
+
+	var conf *netlink.Config
+	if cc.NetNS != 0 {
```

Comment:
As 0 is the zero value of the NetNS field, you can skip this conditional:
```go
return netlink.Dial(unix.NETLINK_NETFILTER, &netlink.Config{NetNS: cc.NetNS})
```

Reply:
Doneskies :]